packages/ui-kit/src/components/Logger/useLog.ts

```diff
@@ -0,0 +1,45 @@
+import { useContext } from 'react'
+
+import { LogConfig, LogLevel } from './Log.types'
+import { LogContext } from './LogProvider'
+
+export const useLog = (): LogConfig => {
+  // Get the log level from the context
+  const context = useContext(LogContext)
+
+  // If the context loglevel is set, use it, otherwise use the default log level as  "error"
+  const level: LogLevel = context.logLevel || LogLevel.Error
```

Comment:
But we define LogContext such that `logLevel` is always present right? We don't need to handle this case (for well-typed code).

If we want to keep this, I suggest at least using `??` instead of `||`:

```suggestion
  const level: LogLevel = context.logLevel ?? LogLevel.Error
```

But I think it would be better to avoid spreading the defaulting logic across this file, and packages/ui-kit/src/components/Logger/LogProvider.tsx